test/integration/model/count.test.js

```diff
@@ -133,5 +133,47 @@ describe(Support.getTestDialectTeaser('Model'), function() {
       });
     });
 
+    it('should be able to use where clause on included models', function() {
+      const queryObject = {
+        col: 'username',
+        include: [this.Project],
+        where: {}
+      };
+      queryObject.where['$Projects.name$'] = 'project1';
+      return this.User.count(queryObject).then((count) => {
+        expect(parseInt(count)).to.be.eql(1);
+        const queryObject = {
+          col: 'username',
+          include: [this.Project],
+          where: {}
+        };
+        queryObject.where['$Projects.name$'] = 'project2';
+        return this.User.count(queryObject);
```

Comment:
Switching to eslint would be nice I guess. The rules there are much more powerful than the jshint ones. And maybe [standard](https://github.com/feross/standard) would also be a good idea to use.